protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepFormToolbox.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react'
+import get from 'lodash/get'
+import { useTranslation } from 'react-i18next'
+
+import {
+  ALIGN_CENTER,
+  Flex,
+  Icon,
+  PrimaryButton,
+  SPACING,
+  StyledText,
+  Toolbox,
+} from '@opentrons/components'
+import { stepIconsByType } from '../../../../form-types'
+import {
+  CommentTools,
+  HeaterShakerTools,
+  MagnetTools,
+  MixTools,
+  MoveLabwareTools,
+  MoveLiquidTools,
+  PauseTools,
+  TemperatureTools,
+  ThermocyclerTools,
+} from './StepTools'
+import type { StepFieldName } from '../../../../steplist/fieldLevel'
+import type { FormData, StepType } from '../../../../form-types'
+import type { FieldPropsByName, FocusHandlers, StepFormProps } from './types'
+
+type StepFormMap = {
+  [K in StepType]?: React.ComponentType<StepFormProps> | null
+}
+
+const STEP_FORM_MAP: StepFormMap = {
+  mix: MixTools,
+  pause: PauseTools,
+  moveLabware: MoveLabwareTools,
+  moveLiquid: MoveLiquidTools,
+  magnet: MagnetTools,
+  temperature: TemperatureTools,
+  thermocycler: ThermocyclerTools,
+  heaterShaker: HeaterShakerTools,
+  comment: CommentTools,
+}
+
+interface StepFormToolboxProps {
+  canSave: boolean
+  dirtyFields: string[]
+  focusHandlers: FocusHandlers
+  focusedField: StepFieldName | null
+  formData: FormData
+  propsForFields: FieldPropsByName
+  handleClose: () => void
+  // TODO: add abiltiy to delete step?
+  handleDelete: () => void
+  handleSave: () => void
+}
+
+export const StepFormToolbox = (props: StepFormToolboxProps): JSX.Element => {
+  const {
+    formData,
+    focusHandlers,
+    canSave,
+    handleClose,
+    handleSave,
+    propsForFields,
+  } = props
+  const { t, i18n } = useTranslation(['shared', 'application'])
```

Comment:
if there is no strong reason, I recommend you change the order.

```suggestion
  const { t, i18n } = useTranslation(['application', 'shared'])
```